react/components/Collapsible/index.js

```diff
@@ -5,6 +5,12 @@ import CaretDown from '../icon/CaretDown'
 import CaretUp from '../icon/CaretUp'
 import { jsFocusVisible } from './styles.css'
 
+const colorMap = {
+  black: 'c-on-base',
```

Comment:
```suggestion
  base: 'c-on-base',
```

it won't necessarily be black depending on the color scheme